_posts/2020-05-07-report-deep-sea-robotics.md

```diff
@@ -0,0 +1,182 @@
+In modern times, the first step to develop a robot is using a simulator. A simulator is a tool that developers can use to design a robot and its application without a physical machine, and then transfer to and test through the physical world, saving both time and cost.  For underwater vehicles, simulation prior to actual operations is critical. Missions with underwater vehicles are even more costly and time consuming due to the special environment, the salinity, depth, and movement of the sea. The loss of a prototype at sea if communication fails is exactly the scenario that scientists and developers try to avoid.  In this study, I create a deep sea robot with consideration of current deep sea robots, different simulation platforms, operating systems, and shapes. 
+
+# Engineering Goals
+This goal of this project is to design a robot that can withstand deep sea conditions and navigate the waters, which can help to make discoveries and better our understanding about the deep sea.
+
+# Materials
+* Computer 
+* Ubuntu Operating System 
+* Blender
+* Gazebo Simulator
+* ROS Melodic
+* Unmanned Underwater Vehicle Simulator
+
+# Background and Related Work
+With no previous experience in simulating robots, I devoted a month and a half of initial research time to installing, differentiating, and understanding necessary programs, Gazebo Simulator, Robot Operating System (ROS), Linux distributions, and 3D software. To use these programs, I had to be able to code in XML, C++, and Python. Gazebo Simulator is an open-source dynamic 3D simulator with the capability to simulate populations of robots in complex indoor and outdoor environments (Gazebo, 2019). Working together with Robot Operating System (ROS) that provides necessary interfaces for the robot, Gazebo has become a powerful and popular robot simulator for the roboticists. It has been used as the simulation platform for many national and international contests, such as DARPA Subterranean Challenge (2018-2021), Virtual RobotX Competition (2019), and the NASA Space Robotics Challenge (2016-2017). However, Gazebo and ROS are only compatible on Unix operating systems, and not Microsoft Windows (the operating system on my computer). Thus, I spent time choosing an operating system compatible with both Gazebo and ROS out of MacOS and Linux Distros (Linux Mint and Ubuntu). The biggest difference between Linux Mint and Ubuntu is user interface and support implementations (Devčić, 2016). After experimenting with Linux Mint and Ubuntu, I opted to use Ubuntu 18.04 which proved more efficient to download and run Gazebo. There is also more documentation in context of robot simulation with Ubuntu.  After setting up my computer, I studied 3D software program that I could use to create meshes that simulate the robot. I ended up using Blender over Maya and 3ds Max, because of its price and compatibility with Ubuntu (Educba, n.d.). Then, came the challenge to design a robot through simulation. 
```

Comment:
```suggestion
With no previous experience in simulating robots, I devoted a month and a half of initial research time to installing, differentiating, and understanding necessary programs; including, [Gazebo](http://gazebosim.org/), the [Robot Operating System (ROS)](https://www.ros.org/), Linux distributions (i.e., Ubuntu), and 3D software. To use these programs, I had to code in XML, C++, and Python.

Gazebo Simulator is an open-source dynamic 3D simulator with the capability to simulate populations of robots in complex indoor and outdoor environments (Gazebo, 2019). Working together with ROS, which provides necessary interfaces for the robot, Gazebo has become a powerful and popular robot simulator for the roboticists. It has been used as the simulation platform for many national and international contests, such as DARPA Subterranean Challenge (2018-2021), Virtual RobotX Competition (2019), and the NASA Space Robotics Challenge (2016-2017).

However, Gazebo and ROS are only compatible on Unix operating systems, and not Microsoft Windows (the operating system on my computer). Thus, I spent time choosing an operating system compatible with both Gazebo and ROS out of MacOS and Linux Distros (Linux Mint and Ubuntu). The biggest difference between [Linux Mint](https://www.linuxmint.com/) and [Ubuntu](https://ubuntu.com/) is user interface and support implementations (Devčić, 2016). After experimenting with Linux Mint and Ubuntu, I opted to use Ubuntu 18.04 which proved more efficient to download and run Gazebo. There is also more documentation in the context of robot simulation with Ubuntu.

After setting up my computer, I studied 3D software program that I could use to create meshes that simulate the robot. I ended up using [Blender](https://www.blender.org/) over Maya and 3ds Max, because of its price and compatibility with Ubuntu (Educba, n.d.). Then, came the challenge to design a robot through simulation. 
```